packages/compass-components/src/components/generative-ai/generative-ai-input.tsx

```diff
@@ -342,9 +343,9 @@ function GenerativeAIInput({
       </div>
       {errorMessage && (
         <div className={errorSummaryContainer}>
-          <ErrorSummary data-testid="ai-error-msg" errors={errorMessage}>
+          <Banner data-testid="ai-error-msg" variant={BannerVariant.Danger}>
             {errorMessage}
```

Comment:
this {errorMessage} here was doing nothing with `ErrorSummary`, moving to a Banner